src/js/controllers/ctrl-media-import.js

```diff
@@ -52,7 +56,12 @@ app.controller('MediaImportCtrl', function ($scope, $window, $timeout) {
 	let _allFiles = []
 
 	const onMediaSelect = (media) => {
-		$window.parent.Materia.Creator.onMediaImportComplete([media])
+		setTimeout(() => {
+			if (IS_HIDDEN_CLICK == false) {
```

Comment:
If this just has to be falsy, `==` is fine, or even better `! isHiddenClick`. If the value has to specifically be `false`, use `===`.